src/main/webapp/register.js

```diff
@@ -14,9 +14,19 @@
 
 function onRegisterLoad() {
   supplyLogoutLink();
+  autofillEmail();
   prepareFormValidation();
```

Comment:
This probably needs to be run after `autofillEmail()`. Since autofillEmail is asynchronous, you can't guarantee that it runs afterward unless you `then()` prepareFormValidation off of it